addons/website/views/snippets/snippets.xml

```diff
@@ -52,6 +52,9 @@
                 <t t-snippet="website.s_carousel" string="Carousel" group="intro">
                     <keywords>gallery, carousel</keywords>
                 </t>
+                <t t-snippet="website.s_pricelist_boxed" string="Pricelist Boxed" group="content">
+                    <keywords>menu, pricing, shop, table, cart, product, cost, charges, fees, tarifs, prices, expenses</keywords>
+                </t>
```

Comment:
The `group` attribute is set correctly, but these lines should be moved below the `<!-- Content group -->` comment 😉

Also, I'd move the snippet way below in the list, close to the current `s_product_catalog` so they are rendered side by side in the modal 😉